Add files via upload

mazaquax opened the pull request.

mazaquax committed:
Add files via upload

illarionovEA reviewed (changes requested):
со случайным блужданием какая-то беда. длина траектории должна быть ровно 100. 
не увидел последнюю задачу на подмножества

mazaquax committed:
Add files via upload

Исправил блуждание, сделал последнюю задачу с случайным разбиением n-элементного множества

mazaquax commented:
Исправил блуждание, сделал последнюю задачу с случайным разбиением n-элементного множества
https://www.codewars.com/users/mazaquax
